test/HealthChecks.AzureServiceBus.Tests/HealthChecks.AzureServiceBus.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Why only net7?

Reply:
It was giving me conflicts when building. Don't remember what, however I felt that it's okay with test projects being net7 only as they are not released. I have both net6 and net7 installed at the moment, but I can look at the detailed conflict when I get home, if you want?

Reply:
I would like to try keep both TFMs.

Reply:
I have reverted the changes, however, I do get this warning on these projects. I had both .net6 and .net7 sdk installed. I also tried removing .net6 sdk, only having .net7 sdk, with no difference. But again, it's just a warning (however annoying I find it 😄).

<img width="1394" alt="Compiler warning unit tests" src="https://user-images.githubusercontent.com/1763448/209451500-e4c43b6f-6500-438c-ada9-aa3e666fb17f.png">

Reply:
I saw it. Not so bad for now. Can be handled later.